### PR TITLE
Update `Difficulties` to follow the osu! new difficulty spectrum

### PR DESCRIPTION
**Pull request by Nivalyx**
An update to the `Difficulties` page to follow the changes that were recently introduced in the [osu!web/#7855](http://github.com/ppy/osu-web/pull/7855#issuecomment-880959644) PR.

image

### CODE REVIEW

**Comment by mcendu** [comment]
Should we document the [details] of the spectrum?

[details]: https://github.com/ppy/osu-web/pull/7855#issuecomment-880959644

**Comment by Nivalyx** [comment]
mmmmmmmm looking at the details attached i don't think that's a good idea tho since the details seem to be too technical/too out of scope in an article that is essentially focused on the merits of `Difficulty` itself 😦

File: wiki/Beatmap/Difficulty/en.md [hunk 1]
```diff
@@ -66,11 +66,13 @@ These are the typical levels of difficulty that a beatmap can fall under. Specif
 - ![](/wiki/shared/diff/expert-m.png) Expert
 - ![](/wiki/shared/diff/expertplus-m.png) Expert+
 
-## Star rating
+## Difficulty and star rating
 
-![Star rating ranges](img/SR-range.png)
+As per 27 July 2021, the osu! website automatically assigns a beatmap's difficulty rating colour based on their star rating according to the following spectrum:
```
Comment (Walavouchey) [inline wiki/Beatmap/Difficulty/en.md:71]:
```suggestion
As of 27 July 2021, the osu! website automatically assigns a beatmap's difficulty rating colour based on their star rating according to the following spectrum:
```

File: wiki/Beatmap/Difficulty/en.md [hunk 2]
```diff
@@ -66,11 +66,13 @@ These are the typical levels of difficulty that a beatmap can fall under. Specif
...
 
-On the website, beatmaps are automatically given a difficulty icon based on these star rating ranges:
+![osu! difficulty rating colour spectrum](img/osu-difficulty-spectrum.png)
+
+Some other instances, such as the [AIMod](/wiki/Beatmap_Editor/AiMod), classifies a beatmap's difficulty into six different level ranges based on their star rating as follows:
```
Comment (Walavouchey) [inline wiki/Beatmap/Difficulty/en.md:75]:
```suggestion
In some instances, such as [AIMod](/wiki/Beatmap_Editor/AiMod), difficulty is classified into six level ranges based on star rating as follows:
```

File: wiki/Beatmap/Difficulty/en.md [hunk 3]
```diff
@@ -79,4 +81,4 @@ On the website, beatmaps are automatically given a difficulty icon based on thes
 - ![](/wiki/shared/diff/expert-o.png) Expert: 5.3★–6.49★
 - ![](/wiki/shared/diff/expertplus-o.png) Expert+: 6.5★ and above
 
-The intended difficulty levels of most beatmaps match the icons they are given, but some deviate significantly. This is why the [ranking criteria](/wiki/Ranking_Criteria) define more precise rules for determining difficulty and evaluating beatmaps spreads.
+It is worth noting however that while the intended difficulty levels of most beatmaps match the colours or level ranges they are given to, some others may deviate significantly due to certain factors. This is why the [ranking criteria](/wiki/Ranking_Criteria) offers up more decisive ground rules to determine a beatmap's actual difficulty level and a beatmapset's actual difficulty spread regardless of their star rating.
```
Comment (Walavouchey) [inline wiki/Beatmap/Difficulty/en.md:84]:
```suggestion
Note that while the intended difficulty levels of most beatmaps match the colours or level ranges they are given, some may deviate significantly due to certain factors. This is why the [ranking criteria](/wiki/Ranking_Criteria) offers more decisive ground rules to determine a beatmap's actual difficulty level and spread regardless of its star rating.
```

**Comment by mcendu** [comment]
This might look better?

spectrum

**Comment by Nivalyx** [comment]
applied everything + replaced the spectrum image with the one that mcendu provided, thanks for checking guys 👍

File: wiki/Beatmap/Difficulty/fr.md [hunk 4]
```diff
@@ -9,6 +9,8 @@ tags:
   - nom de la difficulté
   - classement en étoiles
   - étoiles
+outdated:
+  - true
```
Comment (Walavouchey) [inline wiki/Beatmap/Difficulty/fr.md:13]: these should be `outdated: true` on one line (github on mobile doesn't let me do multiline suggestions..)

**Comment by Nivalyx** [comment]
@Walavouchey oh whoops, missed that one

fixed!

**Review by peppy — APPROVED** [review]

**Pull request merged**